trello_app.py

```diff
@@ -0,0 +1,123 @@
+from flask import Blueprint
+from flask import current_app as app
+from flask import Flask, render_template, request, redirect, url_for
+import config as cf
+from model import Item, ViewModel
+
+import session_items as session
+import requests
+import json
+
+trello_bp = Blueprint( 'trello_bp', __name__)
+
+@trello_bp.route('/', methods=['GET'])
+#@app.route('/')
+def index():
+    board_id = cf.get_trello_board_id()
+    url = f"https://api.trello.com/1/boards/{board_id}/cards"
+    query = cf.get_trello_query()
+    
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+        )
+    
+    cards = json.loads(response.text)
+    items_list = list()
+    for card in cards:
+        status = 'To Do' 
+        if card['idList'] == '5f5a4b008a129438843fcf10':
+            status = 'To Do'
+        elif card['idList'] == '5f5a4b008a129438843fcf11':
```

Comment:
These ids should come from config rather than being hardcoded values. This currently means that your selenium tests can't work properly with the test board.